pages/index.vue

```diff
@@ -450,14 +466,18 @@ export default {
 
       await this.$store.dispatch('user/fetchUser').then(response => {
 
-        if(this.organization.length == 0 || !this.localuser.position || !this.localuser.email){
+        if(this.organization.length == 0 || !this.localuser.position || !this.localuser.email || !this.localuser.name){
           this.dialog = true
         }
         this.localuser = JSON.parse(JSON.stringify(this.person))
         this.userAffiliation = this.organization ? this.organization : []
 
         this.localuser.position = this.localuser.position ? this.localuser.position : ''
         this.userPosition = this.localuser.position
+        if(this.localuser.email) { // If a user's email is already stored in the backend we do not need to verify it, thus we change the submitMessage from 'Verify email' to 'Submit'
```

Comment:
don't we want to check that the user's submitted email matches the authenticated email here too? (like you did above)?

Reply:
This block is executed under Vue's mounted() method, so it executes once the page loads. In this case there is no information being submitted from the user. 

Here we are checking whether the backend has sent us an email (i.e. if the localuser object sent from the backend contains a non-empty email field). If it has, the email must already be authenticated since it is stored in the backend and thus the 'submitMessage' field must no longer say "Verify Email", but "Submit" instead since this email does not need to be verified by the backend.

Reply:
gotcha, thanks